services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/diagnosiskeys/structure/file/TemporaryExposureKeyExportFile.java

```diff
@@ -47,7 +55,8 @@ private TemporaryExposureKeyExportFile(Collection<TemporaryExposureKey> temporar
    */
   public static TemporaryExposureKeyExportFile fromTemporaryExposureKeys(
       Collection<TemporaryExposureKey> temporaryExposureKeys, String region, long startTimestamp, long endTimestamp) {
-    return new TemporaryExposureKeyExportFile(temporaryExposureKeys, region, startTimestamp, endTimestamp);
+    return new TemporaryExposureKeyExportFile(temporaryExposureKeys, region, startTimestamp,
```

Comment:
diff pollution

Reply:
Sorry for that - auto formatter went nuts